examples/dynamic-client/namespaced_custom_resource.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This example demonstrates the following:
+    - Creation of a namespaced scoped custom resource definition (CRD) using dynamic-client
```

Comment:
CRD is always cluster scoped

Reply:
Thank you for pointing. I fixed the example description in both `namespaced_custom_resource.py` & `cluster_scoped_custom_object.py`.